addons/tm_tmf_loadouts/CfgLoadouts.hpp

```diff
@@ -716,6 +716,12 @@ class CfgLoadouts
         category = "Other";
         #include "loadouts\afr_militia.hpp"
     };
+    class AFR_AR
```

Comment:
AFR_ARMY or AFRICAN_ARMY would be more descriptive. I like to keep class and file names identical

Reply:
Huh, seems like I forgot half the word 😆 Will fix